sdk/tests/client/input_selection/foundry_outputs.rs

```diff
@@ -924,7 +917,7 @@ fn melted_tokens_not_provided() {
         token_id,
             found,
             required,
-        }) if token_id == token_id_1 && found == U256::from(0) && required == U256::from(100)));
+        }) if token_id == token_id_1 && found.as_u32() == 0 && required.as_u32() == 100));
```

Comment:
I have the slight feeling this has to be `u32` but I don't understand why :sweat_smile:

Reply:
Not sure what you mean